docs/DeveloperGuide.md

```diff
@@ -605,37 +677,49 @@ testers are expected to do more *exploratory* testing.
     1. Other incorrect delete commands to try: `delete`, `delete x`, `...` (where x is larger than the list size)<br>
        Expected: Similar to previous.
 
-1. _{ more test cases …​ }_
-
 ### Saving data
 
 1. Dealing with missing/corrupted data files
 
-    1. _{explain how to simulate a missing/corrupted file, and the expected behavior}_
+    1. Head over to the folder where your jar file is located and then delete both inventory.json and recipeBook.json within the data folder.
```

Comment:
Small suggestion use "open" instead of "head over to", remove the "then" after the "and"